### PR TITLE
Accessibility

### LINKED ISSUE #31: Check screen reader for ARIA labels

### LINKED ISSUE #32: Add tabbability to all elements needed.

### LINKED ISSUE #33: Check Lighthouse audit

### LINKED ISSUE #34: Add button to go back to bookings.

### PR DESCRIPTION
**Pull request by nzambonivergara**
**What (if any) features are you implementing?**
- Added aria labels where necessary.
- Added a tabindex to room cards.
- Added alternative text to room cards' images.
- Added a button to return to bookings.

**What (if anything) did you refactor?**
- NA

**Were there any issues that arose?**
- NA

**Any other comments, questions, or concerns?**
- I ran a Lighthouse audit and the score was 100.
- I also ran the WAVE Chrome extension and there were zero errors or warnings.

### CODE REVIEW

File: dist/index.html [hunk 1]
```diff
@@ -13,9 +13,12 @@
   <body>
     <header class="header">
       <h1 class="header__main-title">
-        <span class="material-icons header__icon">grass</span>
+        <span class="material-icons header__icon" aria-label="overlook-hotel-logo">grass</span>
```
Comment (CLPena) [inline dist/index.html]: aria label values can be written normally, like "Overlook hotel logo" and don't require kebab casing or anything. I've never actually seen then written in kebab case and am not 100% that a screen reader would properly interpret them that way (though I could be wrong!)

File: dist/index.html [hunk 2]
```diff
@@ -13,9 +13,12 @@
...
         Overlook</h1>
-      <button class="log-out-button">LOG OUT</button>
+      <button class="log-out-button">
+        <p>LOG OUT</p>
```
Comment (CLPena) [inline dist/index.html]: It depends on the screen reader software, but often times words written in all caps will be interpreted as acronyms and read as the letters individually, so "L O G O U T" vs "log out." I believe that using `text-transform` in your CSS is a workaround for this.

File: src/domUpdates.js [hunk 3]
```diff
@@ -44,8 +45,8 @@ const domUpdates = {
     rooms.forEach(room => {
       const bidet = room.hasBidet ? 'yes' : 'no';
       availableRoomsContainer.innerHTML +=
-      `<article class="rooms-container__room-card" id=${room.number}>
-        <img src=${room.imageSrc}>
+      `<article class="rooms-container__room-card" id=${room.number} tabindex="0">
+        <img src=${room.imageSrc} alt="${room.bedSize} bed bedroom">
```
Comment (CLPena) [inline src/domUpdates.js:49]: Does this alt text work? I might double check that your interpolation is doing what you intend here.

**Review by CLPena — COMMENTED** [review]
This looks really good, Natalia! I left a few comments and focused primarily on accessibility-related code to stay within the scope of this PR. Let me know if you want me to look at anything else!

File: dist/index.html (hunk 1 above)
Comment (nzambonivergara) [reply dist/index.html]: Thank you for pointing that out. Fixed those!

File: dist/index.html (hunk 2 above)
Comment (nzambonivergara) [reply dist/index.html]: Good idea! Changed that!

**Pull request merged**